import { AppException } from '../app.exception';

export class PacienteNaoEncontradoException extends AppException {
  constructor() {
    super('Paciente n√£o encontrado!', 404);
  }
}